app/utils/hstasklisthelper/HsTaskListHelperUtils.scala

```diff
@@ -92,6 +104,9 @@ trait HsTaskListHelperUtils extends Enumerable.Implicits {
     case EstablisherCompanyAddress | TrusteeCompanyAddress => messages("messages__schemeTaskList__sectionEstablishersCompany_change_address", _)
     case EstablisherCompanyContactDetails | TrusteeCompanyContactDetails => messages("messages__schemeTaskList__sectionEstablishersCompany_change_contact", _)
     case EstablisherCompanyDirectors => messages("messages__schemeTaskList__sectionEstablishersCompany_change_directors", _)
+    case TrusteeIndividualDetails => messages("messages__schemeTaskList__sectionEstablishersIndividual_change_details",  _)
+    case TrusteeIndividualAddress => messages("messages__schemeTaskList__sectionEstablishersIndividual_change_address",  _)
```

Comment:
can we reuse existing messages for address and contact details by giving them generic names?

Reply:
done